src/tools/run-make-support/src/rustc.rs

```diff
@@ -150,6 +150,20 @@ impl Rustc {
         self
     }
 
+    /// Pass a codegen option.
+    pub fn codegen_option(&mut self, option: &str) -> &mut Self {
+        self.cmd.arg("-C");
+        self.cmd.arg(option);
+        self
+    }
```

Comment:
I think specific helpers for specific codegen options are better, i.e. `.lto()` over `.codegen_options("lto")` or something.